src/cargo/core/workspace.rs

```diff
@@ -527,6 +544,18 @@ impl<'cfg> Workspace<'cfg> {
     }
 }
 
+fn expand_member_path(path: &Path) -> CargoResult<Vec<PathBuf>> {
+    let path = path.to_str().unwrap();
```

Comment:
Could the `None` case be handled here instead of `unwrap`? (returning an empty `Vec`)